### PR TITLE
Feature/rafi/create module completion model and migration

### PR DESCRIPTION
**Pull request by rafipriatna**
Issue: https://trello.com/c/PWeS3N2G/40-be-create-modulecompletions-model-and-migration
Add:
- Model and migration file for `module_completions`

Fix:
- Typo at `create lecture` migration and `Lecture` model

### CODE REVIEW

File: src/database/entities/ModuleCompletion.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    @Column({ name: 'module_id' })
+    moduleId!: number;
+
+    @Column({ name: 'user_id' })
+    userId!: number;
+
+    @Column({ name: 'completion_time' })
+    completionTime!: Timestamp;
+
+    @ManyToOne(() => Module, (module) => module.lectures)
```
Comment (fajartd02) [inline src/database/entities/ModuleCompletion.ts]: ini harusnya `module.moduleCompletions` gak ya?

Comment (fajartd02) [reply src/database/entities/ModuleCompletion.ts]: oh ya sama @OneToMany di modulenya belom di update ya?

**Review by fajartd02 — CHANGES_REQUESTED** [review]

File: src/database/entities/ModuleCompletion.ts (hunk 1 above)
Comment (rafipriatna) [reply src/database/entities/ModuleCompletion.ts]: Fixed

**Review by fajartd02 — APPROVED** [review]
lgtm

**Pull request merged**